src/components/todo-item-card-edit/index.ts

```diff
@@ -0,0 +1,3 @@
+import { TodoItemCardEdit } from "./TodoItemCardEdit";
```

Comment:
Still can't understand why you are doing this.

Reply:
oh, I thought there is a way to export all items in one row
**export * from "./todoItems"** - and i thought ITodoItems, TodoItems, TodoEnums will export. But always get error. That's why I have left so it untouched.